DFC.App.Banners.Services.CacheContentService.UnitTests/DocumentServiceTests/BannerDocumentServiceTests.cs

```diff
@@ -30,15 +35,15 @@ public async Task BannerDocumentServiceDeleteAsyncDeleteByGuidId(bool documentSe
         public async Task BannerDocumentServiceGetByIdAsyncReturnsPageBannerContentItemModels()
         {
             // Arrage
-            A.CallTo(() => FakeDocumentService.GetByIdAsync(A<Guid>.Ignored, A<string>.Ignored)).Returns(BuildValidContentItemModel());
+            A.CallTo(() => FakeDocumentService.GetByIdAsync(A<Guid>.Ignored, A<string>.Ignored)).Returns(BuildValidPageBannerContentItemModel());
```

Comment:
BuildValidPageBannerContentItemModel(), assign it to a variable and instead of building it twice.

Reply:
Here and on line 46